node-module-system/calculate.js

```diff
@@ -0,0 +1,28 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+const [, , ...values] = process.argv;
+const [firstNumericValue, operation, secondNumericValue] = values;
+let result;
+switch (operation) {
+  case 'plus':
+    result = add(Number(firstNumericValue), Number(secondNumericValue));
```

Comment:
Optional, but it might look nicer to do this Number conversion up top. That way this switch could be focused on just the logic, without worrying about getting the proper data from the command line.